src/Compilers/CSharp/Portable/Binder/LocalBinderFactory.cs

```diff
@@ -199,6 +199,45 @@ public override void VisitOperatorDeclaration(OperatorDeclarationSyntax node)
             Visit(node.ExpressionBody);
         }
 
+        public override void VisitInvocationExpression(InvocationExpressionSyntax node)
+        {
+            if (node.MayBeNameofOperator())
+            {
+                WithTypeParametersBinder nextWhenNameofOperator = ((_enclosing.Flags & BinderFlags.InContextualAttributeBinder) != 0) && MessageID.IDS_FeatureExtendedNameofScope.IsFeatureAvailable(node)
```

Comment:
>MessageID.IDS_FeatureExtendedNameofScope.IsFeatureAvailable(node) [](http://example.com/codeflow?start=137&length=65)

Do we really want the lookup to be affected by the language version?

Reply:
Yes, this is intentional. The language version condition will be removed by the time C# 11 ships. Until then the compat breaks that come from `nameof(parameter)` feature will be scoped to preview users. This was discussed in LDM.
I added a bullet to the test plan as a reminder. I can also file an issue. This is something we shouldn't forget indeed.